src/coreclr/jit/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ if(CLR_CMAKE_TARGET_WIN32)
 endif(CLR_CMAKE_TARGET_WIN32)
 
 set( JIT_SOURCES
+  clrjit.natvis
```

Comment:
Someone should verify by picking my changes and making sure that the `clrjit.natvis` file is picked for them.

Reply:
This didn't work for me. I needed to manually copy to my "user" Visualizers directory, `Documents\Visual Studio 2019\Visualizers`

Reply:
You might need to conditionally include this when building on Windows only. I don't know how CMake handles natvis files for non-MSVC compilers.

Reply:
From the CI, it looks like it build fine, so I am inclined to keep it as is. @BruceForstall - did you do a one time build before trying? I honestly don't know how it picks, but after the build, it started picking my local changes that I made to the file in `src\coreclr\jit`.

Reply:
@BruceForstall how are you testing out the visualizations? Are you using the MSBuild generator and opening the project in VS or are you using a VS exe debugging project?

My guess would be that we are including the natvis in the VS project, but we aren't providing the right commands for the Ninja build to embed the natvis into the pdb.

Reply:
I keep getting the following message in `Output` window of VS which lets me believe that it is embedded in `clrjit.pdb` file.

![image](https://user-images.githubusercontent.com/12488060/118055675-65f71280-b33d-11eb-901b-36153069edbe.png)

But yes, as @jkoritzinsky mentioned, I also used `-vs coreclr.sln` to regenerate project files.

Reply:
I'm guessing that the generated MSBuild project is correctly embedding the natvis, but Ninja is not.

Reply:
I build with ninja (default) using "build.cmd", then have a VS C++ project that I use to set debugging configuration. I don't use "build -vs coreclr.sln". So, it sounds like you're right that ninja isn't embedding it.

I'm not sure how much I'm worried about it not getting embedded. Seems like a "nice to have", not a "must have".

Reply:
I think it's worth doing, but we can do it in a follow-up PR if needed.

Reply:
@jkoritzinsky - do you have any suggestion on what needs to be done here?

Reply:
I think you need to manually add the [`/NATVIS` linker flag](https://docs.microsoft.com/en-us/cpp/build/reference/natvis-add-natvis-to-pdb?view=msvc-160) manually to the target and have it point to the (probably full path) of the natvis file.

Reply:
Should that be done regardless of `Checked`, `Debug` or `Release`?

Reply:
I think it's worth doing for all configurations.

Reply:
With the latest changes, I was able to use `clrjit.natvis` inside different repo. @BruceForstall - let me know if new changes automatically picks the natvis file for you.

Reply:
With these changes, I do see the natvis getting picked up from the PDB.

Reply:
nice.